app/models/user.js

```diff
@@ -12,6 +12,7 @@ export default Model.extend({
   email: attr(),
   firstName: attr(),
   lastName: attr(),
+  githubAuthToken: attr(),
```

Comment:
My gut instinct is that we do not want to expose this kind of data to the client. The API should ideally handle any sensitive data from GitHub and determine what's exposed to the client from there. We can render out a `github_id` to the client (and perhaps some other GitHub user information, as well, like username and photo), but the talking between GitHub and our service should happen via the API.

If you think this needs further discussion, we can do so in an issue. I am leaning heavily towards API-first here, though.